### PR TITLE
remove ingroup MODULENAME from file documentation

### PR DESCRIPTION
**Pull request by pirovc**
#160 

Removing \ingroup command only if it has a preceding \file command.

### CODE REVIEW

File: include/seqan3/alphabet/structure/all.hpp [hunk 1]
```diff
@@ -33,7 +33,6 @@
 // ============================================================================
 
 /*!\file
```
Comment (pirovc) [inline include/seqan3/alphabet/structure/all.hpp]: Not sure here if the second \ingroup should also be removed

Comment (marehr) [reply include/seqan3/alphabet/structure/all.hpp]: yes

Comment (h-2) [reply include/seqan3/alphabet/structure/all.hpp]: No, actually not. Because that's not file documentation, but documentation for (sub)Module!

**Comment by h-2** [comment]
Thanks for your PR! Other than reverting the second commit, this looks good!

**Comment by h-2** [comment]
@pirovc Can you finish this soon so we can merge it? Otherwise you will soon get conflicts, because it touches almost every file...

**Comment by pirovc** [comment]
@h-2 sure, done

**Pull request merged**